ds3-autogen-python/src/main/java/com/spectralogic/ds3autogen/python/model/response/StringParsePayload.java

```diff
@@ -0,0 +1,20 @@
+package com.spectralogic.ds3autogen.python.model.response;
+
+import static com.spectralogic.ds3autogen.python.helpers.PythonHelper.pythonIndent;
+
+public class StringParsePayload implements ParsePayload {
+    private final Integer code;
+
+    public StringParsePayload(final int code) {
+        this.code = code;
+    }
+
+    @Override
+    public String toPythonCode() {
+        final StringBuilder builder = new StringBuilder();
+        builder.append("if self.response.status == ").append(code).append(":\n")
```

Comment:
You can put the text that's apart of the first append statement into the constructor of the StringBuilder.

Reply:
did not know that. updating.